### PR TITLE
MATX-329 : adsklib doc cleanup.

### PR DESCRIPTION
**Pull request by kwokcb**

### CODE REVIEW

**Comment by kwokcb** [comment]
@nicolassavva-autodesk . I assume removing the internal adsk reference is okay with you.

**Review by krishnancr — APPROVED** [review]
Looks good to me.

File: libraries/adsk/adsklib/adsk_colorcorrect.mtlx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0"?>
 <materialx version="1.38" xmlns:xi="http://www.w3.org/2001/XInclude">
-  <!-- Sample color correct interface. Versioned and namespaced -->
+  <!--
+  DESCRIPTION: Sample color correct interface. Versioned and namespaced
+  VERSION: 1.0
```
Comment (nicolassavva-autodesk) [inline libraries/adsk/adsklib/adsk_colorcorrect.mtlx:5]: did we decide that it should be version 1.0?

**Review by nicolassavva-autodesk — COMMENTED** [review]
Minor versioning comment otherwise looks good to me

**Pull request merged**